.github/workflows/build-and-test-dotnet.yml

```
@@ -10,8 +10,14 @@ on:
           are used.
 
     inputs:
+      cancel-in-progress-for-this-pr:
+        description: >
+          When set to "true", it will cancel the already running workflow for this PR. See the concurrency settings of
+          the job for more details.
```

Also for all the others. "PR" is not an obvious abbreviation if you're not already thinking in terms of pull requests, and these workflows aren't just executed under PRs.

```suggestion
          When set to "true", it will cancel the already running workflow for this pull request. See the concurrency
          settings of the job for more details.
```